src/pybind/mgr/dashboard/frontend/src/app/ceph/rgw/rgw-bucket-list/rgw-bucket-list.component.html

```diff
@@ -1,5 +1,4 @@
 <cd-table #table
```

Comment:
I think we should use an AlertPanelComponent to display the warning to the user, regarding stale data.
```suggestion
<cd-alert-panel *ngIf="TIME CONDITION IS REACHED"
                type="warning"
                i18n>INFORM THE USER ABOUT POSSIBLE STALE DATA</cd-alert-panel>

<cd-table #table
```